impl/cluster/pom.xml

```diff
@@ -57,6 +58,18 @@
       <version>${dependency.bean-matchers.revision}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.powermock</groupId>
+      <artifactId>powermock-module-junit4</artifactId>
+      <version>${dependency.org.powermock.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.powermock</groupId>
+      <artifactId>powermock-api-mockito</artifactId>
+      <version>${dependency.org.powermock.version}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Please move this to the `dependencyManagement` of the top level POM in this repository.
In the future we may refactor all projects to use the parent POM.

Reply:
Done